src/services/request.service.js

```diff
@@ -30,6 +30,33 @@ class RequestService {
       order: [['status', 'ASC'], ['createdAt', 'DESC']]
     });
   }
+
+  /** find Request
+   * @static
+   * @param {object} property
+   * @memberof AccommodationService
+   * @returns {object} data
+   */
+  static findTypeByProperty(property) {
```

Comment:
WHat if you rename it to findRequestByProperty as you are finding request

Reply:
👍